romulus.xml

```diff
@@ -44204,6 +44208,10 @@
 		<id>TYPE</id>
 		<default>NA</default>
 	</attribute>

```

Comment:
sorry I do not understand. I added opening tag and end tag for the NO_APSS_PROC_POWER_VCS_VIO_WATTS attribute and elements. Could you please let me know which element miss the opening tag?